src/frontend/packages/core/src/shared/components/list/data-sources-controllers/list-data-source-config.ts

```diff
@@ -96,6 +96,11 @@ export interface IListDataSourceConfig<A, T> {
    */
   refresh?: () => void;
 
+  /**
+   * A function that will be called instead of the default update metrics action
+   */
+  customUpdate?: (action: Action) => void;
```

Comment:
Can we use a name that better describes what the method is actually used for? `customUpdate` is a bit ambiguous .